src/indexer/mod.rs

```diff
@@ -159,8 +161,11 @@ where
                     )
                 })
                 .unzip::<_, _, Vec<_>, Vec<_>>();
-            let results = self.eth.batch(queries).await?;
-
+            let results = if !queries.is_empty() {
+                self.eth.batch(queries).await?
+            } else {
+                Vec::new()
+            };
```

Comment:
There seems to have been a bug here that `self.eth.batch(queries)` fails when queries is empty... it only occurred in sqlite though.